src/page/AddressStep.page.ts

```diff
@@ -2,7 +2,7 @@ import { $, ElementFinder, promise } from 'protractor';
 
 export class AddressStepPage   {
   private get proceedToCheckout(): ElementFinder {
-    return $('#center_column > form > p > button > span');
+    return $('#center_column button span');
```

Comment:
I think this has the property `name` while that is available we should prefer that one:
`[name="processAddress"]`

Reply:
Hecho